knowledgeable/Makefile

```
@@ -1,31 +1,53 @@
-.PHONY: help test lint dev-up dev-down dev-build dev-logs setup-hooks install-gosec
+.PHONY: help go-test smoke-test lint dev-up dev-down dev-build dev-logs setup-hooks dev-migrate hooks-run install-gosec swagger
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)